x-pack/libbeat/reader/parquet/parquet_test.go

```diff
@@ -131,12 +132,16 @@ func createRandomParquet(t testing.TB, fname string, numCols int, numRows int) m
 	// creates an Arrow memory pool for managing memory allocations
 	memoryPool := memory.NewGoAllocator()
 
+	// uses a timestamp as the seed for generating random data
+	seed := time.Now().UnixNano()
+	r := rand.New(rand.NewSource(seed))
```

Comment:
Please use a seed of 1; we want the tests to be reproducible.

Reply:
@efd6 made the changes